### ISSUE TITLE
Optimize performance of this page

### ISSUE THREAD

**Issue by dadiorchen** [Greenstand/treetracker-query-api#254]
These API cause the loading of page below pretty slow:

API took: /trees?organization_id=178&limit=20&offset=0 11775.996934
API took: /trees?organization_id=178&limit=20&offset=0 11481.49868

![Image](https://user-images.githubusercontent.com/5744708/200106255-3bf2950e-4b60-4810-a6c0-5fe5f4430158.png)

**Comment by Mohmn** [comment]
i think the performance of the api for this endpoint(https://prod-k8s.treetracker.org/query/trees?organization_id=178&limit=20&offset=0)
 is good enough @dadiorchen

**Comment by Mohmn** [comment]
![image](https://user-images.githubusercontent.com/37132584/200144250-fc33602f-fbf0-4d3a-ade6-839fa52ee35c.png)

**Comment by Mohmn** [comment]
![image](https://user-images.githubusercontent.com/37132584/200144281-e9c1632d-8a05-4e48-a932-dc4aa953ffa1.png)

**Comment by dadiorchen** [comment]
No, @Mohmn the request randomly gets slow. I also observed fast responses like yours, but sometimes, it takes 12s+

**Comment by dadiorchen** [comment]
So it's getting more tricky to analyse, let me check if I can let you visit the grafana for more info
